README update (#1)
WATERVERSE/WATERVERSE-DCAT-AP-Catalogue: README.md

# WATERVERSE-DCAT-AP-Catalogue

> WATERVERSE Datasets, Entity Data &amp; Data Services **metadata** definitions based on the **DACAT-AP** standard. 

Reviewer: ```suggestion
WATERVERSE Datasets, Entity Data &amp; Data Services **metadata** definitions based on the **DCAT-AP** standard. 
```

# Catalogue 

|           | Name    | Data Model URL | Metadata DCAT-AP URL |
|-----------|---------|----------------|----------------------|
| **Entity**    | [Entity Name] | [Link]         | [Link]               |
| **Dataset**   | [Dataset Name] | [Link]         | [Link]               |
| **Data Service** | [Service Name] | [Link]     | [Link]               |

# DCAT-AP data model

> The DCAT-AP standard will be used to define the metadata of different Entities, datasets, and data services, ensuring structured and interoperable descriptions for each of them. 

Reviewer: ```suggestion
The DCAT-AP standard is used to define the metadata of different Entities, Datasets, and Data Services, ensuring structured and interoperable descriptions for each of them. 
```

Link to the DCAT-AP datamodel: 
https://github.com/smart-data-models/incubated/blob/master/metadata/schema.json

# Project structure 

> The project will be organized into three folders that align with the requirements of WATERVERSE Pilot projects:

Reviewer: ```suggestion
The project is organized into three folders that align with the requirements of WATERVERSE Pilot projects:
```

-  **Datasets**
-  **Entity Data**  
-  **Data Services** 

> For each Datasets, Entity Data &amp; Data Services, there will be two folders per pilot corresponding to:

Reviewer: ```suggestion
For each Datasets, Entity Data &amp; Data Services, there are two folders per pilot corresponding to:
```
